docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc

```diff
@@ -484,6 +486,159 @@ spec:
 Please note that the environment variables related to policy selection mentioned in the Elastic Agent link:https://www.elastic.co/guide/en/fleet/current/agent-environment-variables.html[docs] like `FLEET_SERVER_POLICY_ID` will be managed by the ECK operator.
 
 
+[id="{p}-elastic-agent-running-as-a-non-root-user"]
+// tag::configuration-example-elastic-agent-running-as-a-non-root-user[]
+=== Running as a non-root user
+
+In order to run Elastic Agent as a non-root user you must choose how you want to persist data to the Agent's volume.
+
+1. Run Elastic Agent with an `emptyDir` volume. This has the downside of not persisting data between restarts of the Elastic Agent which can duplicate work done by the previous running Agent.
+2. Run Elastic Agent with a `hostPath` volume in addition to a `Daemonset` running as `root` that maintains permissions for the `agent` user.
```

Comment:
```suggestion
2. Run Elastic Agent with a `hostPath` volume in addition to a `Daemonset` running as `root` that sets up permissions for the `agent` user.
```
"maintain" implies it will constantly reconcile the permissions which it will not iiic